sdk/servicebus/Microsoft.Azure.ServiceBus/src/IClientEntity.cs

```diff
@@ -44,6 +44,11 @@ public interface IClientEntity
         /// </summary>
         ServiceBusConnection ServiceBusConnection { get; }
 
+        /// <summary>
+        /// Returns true if connection is owned and false if connection is shared.
+        /// </summary>
+        bool OwnsConnection { get; }
```

Comment:
For context, this property exists on the `ClientEntity` base class, but was not exposed via the interface.  Unless I overlooked something, this does not appear to require any further implementation on types exposing the interface.

The breaking change is to the interface contract only.

Reply:
That's correct. `OwnsConnection` was added to all `IClientEntity` implementors and not the interface itself to allow the fix in a minor release. This issue will complete the implementation by extracting it into the contract, while still having the actual implementation on place. Hence the breaking change.